bindings/cs/MongoDB.Libmongocrypt/MongoDB.Libmongocrypt.targets

```diff
@@ -1,10 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-
-    <ItemGroup Condition="$([MSBuild]::IsOsPlatform('Windows'))">
-        <Content Include="$(MSBuildThisFileDirectory)../x64/native/windows/mongocrypt.dll">
-            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-            <Link>mongocrypt.dll</Link>
-        </Content>
-    </ItemGroup>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Not sure about this change. I remember that we definitely had a case where we needed exactly the previous structure. But I wasn't able to recall it for now. 
So, I will approve it